backend/app/api/v1/endpoints/cpt/maps/hce.py

```diff
@@ -15,23 +15,26 @@
 #   "version"
 #   "testName"
 ################################################################
-async def hceMapper(start_datetime: date, end_datetime: date):
-    df = await getData(start_datetime, end_datetime, f'hce.elasticsearch')
-    if len(df) == 0:
-        return df
-    df["releaseStream"] = "Nightly"
-    df["ciSystem"] = "Jenkins"
-    df["testName"] = df["product"] + ":" + df["test"]
-    df["product"] = df["group"] 
-    df["jobStatus"] = df['result'].apply(lambda x: "SUCCESS" if x == 'PASS' else "FAILURE")
-    df["version"] = df['version'].apply(lambda x: x if len(x.split(":")) == 1 else x.split(":")[1][:7])
-    df["uuid"] = df["result_id"]
-    df["buildUrl"] = df["link"]
-    df["startDate"] = df["date"]
-    df["endDate"] = df["date"]
-    df = dropColumns(df)
-    return df
-
+import pandas as pd
+async def hceMapper(start_datetime: date, end_datetime: date, size: int, offset: int):
+    response = await getData(start_datetime, end_datetime, size, offset, f'hce.elasticsearch')
+    if response:
+        df = response["data"]
+        if len(df) == 0:
+            return df
+        df["releaseStream"] = "Nightly"
+        df["ciSystem"] = "Jenkins"
+        df["testName"] = df["product"] + ":" + df["test"]
+        df["product"] = df["group"] 
+        df["jobStatus"] = df['result'].apply(lambda x: "SUCCESS" if x == 'PASS' else "FAILURE")
```

Comment:
And, as a general maintainability/readability suggestion to the maintainers: after all the outstanding PRs are merged and stabilized, it'd be a great idea to
1. run `black` and `Prettier` on all Python and JavaScript/CSS files, and
2. add a CI lint check GitHub Action that'll fail a PR if it breaks the style rules. (Of course adding additional lint checks with `flake8`/ES lint/CodeQL would be a great idea too.)
3. Submit this as a PR with no *functional* changes; it might be large, (depending on how bad the code is), but it'd be straightforward to review.

Reply:
I hope that rebasing all of the outstanding PRs against the formatting changes in #139 won't cause major pain ... but it'll be nice to have the reformatting out of the way. I plan to add `black --check` to the CI; probably along with `flake8`.